pkgs/top-level/all-packages.nix

```diff
@@ -26722,13 +26727,17 @@ with pkgs;
   };
   systemdStage1 = systemdMinimal.override {
     pname = "systemd-stage-1";
+    withAcl = true;
     withCryptsetup = true;
     withFido2 = true;
+    withKmod = true;
+    withPam = true;
```

Comment:
I assume we care about ACL and kmod in stage1, but I am not sure about the PAM stack. Can it be dropped?

Reply:
I'm not sure. It's probably needed to `ssh` in (?), as well as to enter the root password in case there was an error? @ElvishJerricco, any thoughts?

Reply:
To be honest I'm not entirely sure. We shouldn't need PAM, as we don't include anything else related to PAM (for instance, the ssh module in #169116 has `UsePAM no`). I would wager that kmod is needed for `systemd-modules-load`, in which case yes that is a requirement for systemd stage 1. But I'm not sure about `withAcl`.

Reply:
Thanks @ElvishJerricco!

If we aren't using PAM this early, I am happy to drop it and re-test relevant things.

ACL I think is needed. I am not deeply familiar with NixOS yet (I have just recently started playing around with it), but a lot of modern system use ACL to allow access to `/dev/dri/` device nodes, and that's something Plymouth for example takes advantage of IIRC.